### PR TITLE
Create new organizers with default `is_signee: false`

### PR DESCRIPTION
**Pull request by rluodev**
## Summary of the problem
We were creating new organizers and organizer position invites with `is_signee: null` because we didn't have a default value.

## Describe your changes
This will set the default value for newly created `organizer_position`s to `false` instead of `null`

### CODE REVIEW

**Review by sarthaktexas — DISMISSED** [review]
this looks good to me but await gary review

**Comment by garyhtou** [comment]
@sampoder would you be interested in reviewing and running the migration on your computer?

**Comment by garyhtou** [comment]
🪄 after Sam's review

File: app/services/event_service/create_demo_event.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def run
 
         @event.save!
 
-        OrganizerPositionInviteService::Create.new(event: @event, sender: @point_of_contact, user_email: @email, initial: true).run!
+        OrganizerPositionInviteService::Create.new(event: @event, sender: @point_of_contact, user_email: @email, initial: true, is_signee: true).run!
```
Comment (garyhtou) [inline app/services/event_service/create_demo_event.rb]: We can't guarantee that the creator of a demo mode ends up signing the contract. The ops person who onboards and approves an org should also update contract signee status.

```suggestion
        OrganizerPositionInviteService::Create.new(event: @event, sender: @point_of_contact, user_email: @email, initial: true).run!
```

**Review by sampoder — APPROVED** [review]
Migration ran on my computer, thanks Ruien!

**Pull request merged**